{{cookiecutter.project_slug}}/HOWTOBAKE.md

```diff
@@ -0,0 +1,45 @@
+# {{ cookiecutter.project_slug }} - How to Bake your Cookie
+
+{{ cookiecutter.project_short_description }}
```

Comment:
how to bake? :-)  you should rather call it "cookies recipes" :-)

Reply:
;)